### ISSUE TITLE
adjustSliderToPosition not working on @react-native-community/slider package

### ISSUE THREAD

**Issue by Noitham** [wix/Detox#3226]
### What happened?

When using the `adjustSliderToPosition` on the new `@react-native-community/slider` package target view is not matched, error log here:

```
 DetoxRuntimeError: Test Failed: Action will not be performed because the target view does not match one or more of the following constraints:
    (is assignable from class <class com.facebook.react.views.slider.ReactSlider> and (view has effective visibility <VISIBLE> and view.getGlobalVisibleRect() to return non-empty rectangle))
    Target view: "ReactSlider{id=1453, visibility=VISIBLE, width=942, height=50, has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=true, is-layout-requested=false, is-selected=false, layout-params=android.view.ViewGroup$LayoutParams@c0b0000, tag=remeasurement-slider, root-is-layout-requested=false, has-input-connection=false, x=41.0, y=396.0}"
```
Component attributes:
```
 attributes: {"identifier":"remeasurement-slider","visibility":"visible","visible":true,"alpha":1,"elevation":0,"height":50,"width":942,"focused":false,"enabled":true,"value":2}

```

If using the [deprecated](https://reactnative.dev/docs/slider) `react-native` package Slider, it works.

Should support be added to the new community package? On iOS it works.

### What was the expected behaviour?

`adjustSliderToPosition` function would recognise the `@react-native-community/slider` component view on Android.

### Was it tested on latest Detox?

- [X] I have tested this issue on the latest Detox release and it still reproduces.

### Did your test throw out a timeout?

- [X] I have followed the instructions under [Identifying which synchronization mechanism causes us to wait too much](https://wix.github.io/Detox/docs/troubleshooting/synchronization#identifying-which-synchronization-mechanism-causes-us-to-wait-too-much).

### Help us reproduce this issue!

Attempt using `adjustSliderToPosition` on the `Slider` component from the community package on Android.

```"@react-native-community/slider": "4.2.0"``` on Android.

### Detox version

19.4.5

### React Native version

0.66.3

### Detox logs

<details><summary>Detox logs</summary>

```
paste logs here!
```

</details>

### Device logs

<details><summary>Device logs</summary>

```
paste logs here!
```

</details>

**Comment by d4vidi** [comment]
IIRC the community slider is used regularly in our regression project.

>is assignable from class <class com.facebook.react.views.slider.ReactSlider>
That seems suspicious. What's the test code like?

**Comment by Noitham** [comment]
> 

Yep, that's what I think the issue is, the community class name is `com.reactnativecommunity.slider.ReactSlider`,

Test code nothing specific, as soon as using the function `adjustSliderToPosition` and passing the Slider id from the community package, we'll get the error.

**Comment by d4vidi** [comment]
Indeed, we should be able to support both packages. Do you think it'd be possible to get some contribution on this? I think that with some reflection logic, this can be solves fairly easily.
